samples/observability-traces.js

```diff
@@ -0,0 +1,139 @@
+/*!
+ * Copyright 2024 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// sample-metadata:
+//   title: Observability (Tracing) with OpenTelemetry
+//   usage: node observability-traces.js <PROJECT-ID> <INSTANCE-ID> <DATABASE-ID>
+
+'use strict';
+
+// Setup OpenTelemetry and the trace exporter.
+// [START spanner_trace_and_export_spans]
```

Comment:
Please remove this [START] and [END] tags. These tags needs to be coordinated with documentation if we need to publish this in public documentation